Mathlib/Topology/ProperMap.lean

```diff
@@ -226,6 +285,41 @@ theorem isProperMap_iff_isClosedMap_and_compact_fibers :
     rw [← clusterPt_lift'_closure_iff]
     exact hx.mono inf_le_left
 
+/-- An injective and continuous function is proper if and only if it is close. -/
+lemma isProperMap_iff_isClosedMap_of_inj (f_cont : Continuous f) (f_inj : f.Injective) :
+    IsProperMap f ↔ IsClosedMap f := by
+  constructor
```

Comment:
Same here, if at some you have more than one goal it's better (for readability) to use `·`: since the first one is very short, you can do something like
```lean
· ... --short proof
... --longer proof